sv/SDVTrafficState.py

```diff
@@ -90,6 +90,30 @@ def to_primitives(self):
         my_copy.intersecting_lanelets = [ [ll.id, 0 ] for ll, sl in self.intersecting_lanelets ]
         return my_copy
 
+class OccupancyZone:
+    def __init__(self, proximity_func, search_func):
+        self.closest  = partial(self.find, proximity_func, search_func)
+        self.vehicles = []
```

Comment:
perhaps, we should extend this to have `self.pedestrians` as well.... Or maybe two grids? One for vehicles and one for pedestrians? What if we add bicycles in the future?